app/src/main/java/com/github/houseorganizer/houseorganizer/util/LoginHelpers.java

```diff
@@ -0,0 +1,63 @@
+package com.github.houseorganizer.houseorganizer.util;
+
+import android.widget.TextView;
+
+import com.github.houseorganizer.houseorganizer.R;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class LoginHelpers {
+
+    public enum RegisterError {
+        INVALID_PASSWORD,
+        INVALID_EMAIL,
+        EMAIL_USED,
+        INPUTS_EMPTY
+    }
+
+    public static boolean inputsEmpty(String email, String password) {
+        return email.isEmpty() || password.isEmpty();
+    }
+
+    public static boolean isValidEmail(String email) {
+
+        // Regex to check valid email.
+        String regex = "^[\\w!#$%&'*+/=?`{|}~^-]+(?:\\.[\\w!#$%&'*+/=?`{|}~^-]+)*@(?:[a-zA-Z0-9-]+\\.)+[a-zA-Z]{2,6}$";
```

Comment:
We might want to change that regex as I also implemented one in the EditHouse activity

Reply:
maybe this one is better since its the official one but I will test what works with firebase in the refactoring task and create a single function in Util